_config.yml

```diff
@@ -0,0 +1,12 @@
+# Dependencies
+markdown: kramdown
+highlighter: rouge
+
+include: ['docs/']
+
+# Setup
+url: https://ciscocloud.github.io/vaquero-docs
```

Comment:
Just change this to: `url: /vaquero-docs` and the links should work locally as well, tested this on https://fattarsi.github.io/vaquero-docs/docs/current/getting-started.html